src/components/ProgramRecordSendModal/SendLearnerRecordModal.scss

```diff
@@ -15,6 +15,12 @@
 
         > div {
             width: 100%;
+            @extend .d-flex;
+
+            input[type=checkbox] {
```

Comment:
Are you sure that `SelectableBox`  isn't causing this misalignment problem? If it is, we should fix it in Paragon, not CSS in an individual instance.

Reply:
Yes, you are right - the same problem in paragon https://paragon-openedx.netlify.app/components/selectablebox/#as-checkbox